Packs/EWS/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### O365 - Security And Compliance - Content Search v2
+- Fixed an issue where Basic Authentication was still used.
+- Added the *Connection URI* and *AzureADAuthorizedEndpointURI Base* parameters to support GCC High and DoD instances.
+- Certificate Based Authentication is no longer the preferred method of authentication.
```

Comment:
let's talk if we want this flow to be deleted from the integration

Reply:
We may need to. I don't think there is a way to apply the changes we did to an app